tests/api/id/[form]/bearer.test.js

```diff
@@ -252,15 +255,17 @@ describe("/id/[form]/bearer", () => {
 
       expect(res.statusCode).toBe(404);
       expect(JSON.parse(res._getData()).error).toBe("Not Found");
-      expect(logMessageDebugSpy.mock.calls.length).toBe(1);
-      expect(logMessageDebugSpy.mock.calls[0][0]).toBe(
-        "A bearer token was attempted to be created for form 1 by user Admin user but the form does not exist"
-      );
+      expect(
```

Comment:
Is this really more efficient? This takes 6 lines as opposed to the previous 4 and at least to me the first option is more readable

Reply:
It's not about efficiency but that the original statement checked an exact position in the array.  If in future code changes a `logMessage.debug()` is added somewhere before this checked call, like in an additional middleware, this check will fail.  The `checkLogs` function instead looks for the exact expected message in the logMessage array which removes the importance of knowing its exact position.